### PR TITLE
Dynamic html-component width

### PR DESCRIPTION
**Pull request by lxgreen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,19 @@
 
 ## [Unreleased]
 
+### :rocket: New Feature
+
+- `common`
+  - [#179](https://github.com/wix-incubator/rich-content/pull/179) `mapStoreDataToButtonProps` API exposed (to be used in toolbar button structures)
+  - [#179](https://github.com/wix-incubator/rich-content/pull/179) `width` and `height` props added to the plugin config
```
Comment (bchmn) [inline CHANGELOG.md]: is this a change to `common`?

File: packages/plugin-html/src/HtmlComponent.jsx [hunk 2]
```diff
@@ -1,41 +1,84 @@
...
+        this.props.componentData.config.height = this.props.settings.height;
+      } else {
+        this.props.componentData.config.height = INIT_HEIGHT;
+      }
+    }
+  }
+
+  render() {
+    const { styles, props } = this;
+    validate(props.componentData, schema);
```
Comment (bchmn) [inline packages/plugin-html/src/HtmlComponent.jsx]: not specific to this PR but perhaps we should skip validation in production? WDYT?

File: CHANGELOG.md (hunk 1 above)
Comment (lxgreen) [reply CHANGELOG.md]: yup, the values are used in [baseComponent](https://github.com/wix-incubator/rich-content/blob/57d28d0b262849f7265d4d0d6b0976d1789188ba/packages/common/src/Base/createBaseComponent.jsx#L238)

File: packages/plugin-html/src/HtmlComponent.jsx (hunk 2 above)
Comment (lxgreen) [reply packages/plugin-html/src/HtmlComponent.jsx]: hmm good idea. usually all the data changes start when we work on plugin, and it is redundant in production, as data and schema are aligned by then. agree to skip in prod

**Pull request merged**